Sources_v3/StreamChatUI/ChatChannelList/ChatUnreadCountBubbleView.swift

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatUnreadCountBubbleView: UIView {
+    // MARK: - Properties
+    
+    public var inset: CGFloat = 3
+    
+    override open var intrinsicContentSize: CGSize {
+        let height: CGFloat = max(unreadCountLabel.font.pointSize + inset * 2, frame.height)
+        let width = max(unreadCountLabel.intrinsicContentSize.width + inset * 2, height)
+        return .init(width: width, height: height)
```

Comment:
Do we need all of this?
- what if we just change `ChatUnreadCountBubbleView` layout margins and pin the label to those margins? 
- we will invalidate the content size when the value changes and call `setNeedsLayout()`. Wouldn't intrinsic content size work out of the box then?